app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,59 @@
+class OrdersController < ApplicationController
+  before_action :set_order, only: %i[show edit update destroy]
+  before_action :set_customers, only: %i[new edit]
+
+  def index
+    @orders = Order.all
+  end
+
+  def show
+    @order = Order.find(params[:id])
+    @customer = @order.customer
+  end
+
+
+  def new
+    @order = Order.new
+  end
```

Comment:
I see the action here, but when I go to its page, it fatally errors because there's no template associated with it:

![Screenshot 2024-09-07 at 10 35 25 PM](https://github.com/user-attachments/assets/f3a0c8f8-56bb-457b-978c-9bc5ec78f4d6)